connectivity/tests/pod.go

```diff
@@ -108,7 +136,11 @@ func curlEndpoints(ctx context.Context, s check.Scenario, t *check.Test,
 				"X-Very-Secret-Token": "42",
 			})
 			t.NewAction(s, epName, client, ep).Run(func(a *check.Action) {
-				a.ExecInPod(ctx, curl(ep, "-H", "X-Very-Secret-Token: 42"))
+				opts := make([]string, 0, len(curlOpts)+2)
```

Comment:
+2 or +1 :thinking:

Reply:
`"-H", "X-Very-Secret-Token: 42"` are two args I think :thinking:

Reply:
and that is why there are three types of people in the world, those can count, and those can't :sweat: